elixir/apps/web/lib/web/live/resources/edit.ex

```diff
@@ -71,6 +73,7 @@ defmodule Web.Resources.Edit do
       attrs
       |> map_filters_form_attrs()
       |> map_connections_form_attrs()
+      |> maybe_delete_connections(socket.assigns.params)
```

Comment:
`connections` are required on the changeset so I just drop the map key leaving them unchanged, should not be a problem for traffic filters to do something similar (unchanged on edit and drop the key on create).